docs/docs/getting-started/first-steps.md

```diff
@@ -45,9 +43,7 @@ If you encounter any problem with the following steps, make sure to use the [lat
 
     </tabs>
 
-2. Create your [IAM configuration](../workflows/config.md#creating-an-iam-configuration)
-
-    After you created your constellation configuration file, it's time to create your [IAM configuration](../workflows/config.md#creating-an-iam-configuration).
+2. Create your [IAM configuration](../workflows/config.md#creating-an-iam-configuration).
```

Comment:
all steps should a have period or none

Reply:
MS style guide says: "Don’t use a period at the end of list items unless they’re complete sentences, even if the complete sentence is very short."
I think this translates to: Use a period at the end of list items if they’re complete sentences, even if the complete sentence is very short.
So I'd suggest adding periods to steps 4 and 5, too.